### PR TITLE
More design and User Interaction

### PR DESCRIPTION
**Pull request by jackdavidweber**
## Pair programming session with @cjoshea9 
Screen Shot 2020-07-09 at 4 12 25 PM

#### Design Changes
- Use code font for input and output text fields
- Create a clearer distinction between text fields
#### User Interaction Changes
- Immediately translate whenever a user switches input/output language rather than waiting for next key stroke

[Before Demo Video
](https://drive.google.com/file/d/1LVPVSDJm1jmw6niu1P2_KF27M-lgqrxm/view?usp=sharing)[After Demo Video
](https://drive.google.com/file/d/1_RYT03Mg2pMaZ6PLu9-2OQdmyng-8-2A/view?usp=sharing)

### CODE REVIEW

File: src/components/TranslateBoxes.js [hunk 1]
```diff
@@ -2,39 +2,45 @@ import React from 'react';
...
-                    value={output}
-                    disabled
-                    />
+                    <Box className = {classes.box} bgcolor="grey.300">
+                        <TextField
+                        id="standard-multiline-static"
+                        label="Translation"
+                        multiline
+                        fullWidth
+                        // variant="filled"
```
Comment (Danish21) [inline src/components/TranslateBoxes.js]: remove comment?

**Review by Danish21 — APPROVED** [review]
Looks great!

File: src/components/TranslateBoxes.js [hunk 2]
```diff
@@ -2,33 +2,45 @@ import React from 'react';
...
-                    <TextField
-                    id="standard-multiline-static"
-                    label="Translation"
-                    multiline
-                    fullWidth
-                    InputProps={{ disableUnderline: true }}
-                    value={output}
-                    disabled
-                    />
+                    <Box className = {classes.box} bgcolor="grey.300">
```
Comment (g-harel) [inline src/components/TranslateBoxes.js]: Consider making the text darker too to preserve contrast.

**Review by g-harel — APPROVED** [review]

File: src/components/TranslateBoxes.js (hunk 2 above)
Comment (jackdavidweber) [reply src/components/TranslateBoxes.js]: Seems to be more involved change than I thought so made the background color lighter and added a TODO

**Review by cjoshea9 — APPROVED** [review]

**Pull request merged**